pkg/virt-controller/watch/vm_test.go

```diff
@@ -333,6 +334,12 @@ var _ = Describe("VirtualMachine", func() {
 			mockQueue.Wait()
 		}
 
+		execute := func(vm *virtv1.VirtualMachine) {
```

Comment:
Nit: the name `execute()` is confusing, since vm.go already has one. Maybe `sanityExecute()` or something like that would be a better name?

Reply:
Done, thanks!